ignite/metrics/binary_accuracy.py

```diff
@@ -9,6 +9,7 @@ class BinaryAccuracy(Accuracy):
     Note: This metric is deprecated in favor of Accuracy.
     """
     def __init__(self, *args, **kwargs):
+        super(BinaryAccuracy, self).__init__(*args, **kwargs)
         warnings.warn("The use of ignite.metrics.BinaryAccuracy is deprecated, it will be "
                       "removed in 0.2.0. Please use ignite.metrics.Accuracy instead.", DeprecationWarning)
         super(Accuracy, self).__init__(*args, **kwargs)
```

Comment:
There are two calls of `super`, this one to remove ?

Reply:
Done!